xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -982,6 +997,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Indicates buttons height (in centimeter)</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="GroundMarkalignedWithButton" type="xsd:boolean" minOccurs="0">
```

Comment:
We already have that in the *LiftButtonsGroup* above.

Reply:
Then this should be reconsiliated with master.